src/mongo/delegates/Tables.class.php

```diff
@@ -87,7 +87,7 @@ public function __construct($storeName,\MongoCollection $collection,$defaultCont
 
     /**
      * Receive update from subject
-     * @param ImpactedSubject
+     * @param \Tripod\Mongo\ImpactedSubject
      * @return void
      */
     public function update(ImpactedSubject $subject)
```

Comment:
That's true - introducing namespace where there were none (especially since we haven't finished working that out completely) made it easier to use ` use ` in some places.

I will probably change them all to fully qualified namespaces prior to merging this feature branch to master.